framework/src/timesteppers/IterationAdaptiveDT.C

```diff
@@ -244,6 +273,10 @@ IterationAdaptiveDT::computeDT()
     dt *= _growth_factor;
     if (dt > _dt_old * _growth_factor)
       dt = _dt_old * _growth_factor;
+    if (_verbose)
+      _console << "Growing dt based on growth factor (" << _growth_factor
```

Comment:
I haven't followed the unmodified parts of the code, but I assume you have verified that `_dt_old` at this point is  dt _before_ the post sync step?

Reply:
(just had to unfold like 3 lines to see that :-D)